### DISCUSSION TITLE
code to create custom dataset for graph classification and node classification.

### DISCUSSION THREAD

**Discussion by anujsharma25** [pyg-team/pytorch_geometric#8195] [Q&A] [answered]
How to code for create custom dataset for graph classification and node classification?
will be helpful if any source code for both tasks will be shared. Thanks in advance.

**Comment by EdisonLeeeee** [comment, marked as answer]
FYI: 
+ Node classification dataset (typically only one graph existed): https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/datasets/planetoid.py
+ Graph classification dataset (many graphs existed): https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/datasets/tu_dataset.py

**Reply by anujsharma25** [reply]
Thankyou
